projects/schematics/src/migrations/mechanism/rename-symbol/rename-symbol_spec.ts

```diff
@@ -0,0 +1,116 @@
+/// <reference types="jest" />
+
+import { getSystemPath } from '@angular-devkit/core';
+import { TempScopedNodeJsSyncHost } from '@angular-devkit/core/node/testing';
+import { HostTree, Tree } from '@angular-devkit/schematics';
+import {
+  SchematicTestRunner,
+  UnitTestTree,
+} from '@angular-devkit/schematics/testing';
+import * as shx from 'shelljs';
+import { runMigration, writeFile } from '../../../shared/utils/test-utils';
+
+const MIGRATION_SCRIPT_NAME = 'migration-v4-rename-symbol-01';
+
+const fileWithSimpleImport = `import { OtherComponent1, Test1Component } from "@spartacus/storefront";
+import { Test2Component } from "@spartacus/core";
+
+const array = [OtherComponent1, Test1Component, Test2Component];`;
+
+// -----------------------------------------------------------------------
+
+const fileWithSimpleImportWithAlias = `import { OtherComponent2 as Test, TestComponent } from "@spartacus/storefront";
+import { Test2Component } from "@spartacus/core";
+
+const array = [Test, Test1Component, Test2Component];`;
+
+// -----------------------------------------------------------------------
+
+const fileWithSimpleImportAndRename = `import { OtherComponent3, TestComponent } from "@spartacus/storefront";
```

Comment:
Same as above: Is it intentional that `TestComponent` is not used in the array below?